### PR TITLE
Feature/si 2476 require definition id on combined vehicle synthetic mint

### PR DESCRIPTION
**Pull request by LorranSutter**

### CODE REVIEW

File: contracts/implementations/MultipleMinter.sol [hunk 1]
```diff
@@ -146,4 +146,120 @@ contract MultipleMinter is
...
+            abi.encode(
+                MINT_VEHICLE_WITH_DD_TYPEHASH,
+                data.manufacturerNode,
+                data.owner,
+                keccak256(bytes(data.deviceDefinitionId))
+            )
+        );
+
+        if (
+            !Eip712CheckerInternal._verifySignature(
```
Comment (elffjs) [inline contracts/implementations/MultipleMinter.sol:223]: I'm curious—why do we verify second?

File: contracts/implementations/MultipleMinter.sol [hunk 2]
```diff
@@ -146,4 +146,120 @@ contract MultipleMinter is
...
+            newTokenIdVehicle,
+            data.syntheticDeviceAddr,
+            data.owner
+        );
+
+        if (data.attrInfoPairsDevice.length > 0)
+            _setInfos(newTokenIdDevice, data.attrInfoPairsDevice);
+        // ----- END Synthetic Device mint and attributes -----
+
+        // ----- Internal contract state change -----
```
Comment (elffjs) [inline contracts/implementations/MultipleMinter.sol:246]: Another thing I thought of—is the DD something we want to "clear out" upon burn?

**Review by elffjs — APPROVED** [review]

File: contracts/implementations/MultipleMinter.sol (hunk 1 above)
Comment (LorranSutter) [reply contracts/implementations/MultipleMinter.sol:223]: No particular reason. I was just following the other function order.

File: contracts/implementations/MultipleMinter.sol (hunk 2 above)
Comment (LorranSutter) [reply contracts/implementations/MultipleMinter.sol:246]: hummm good point. On it

**Pull request merged**